compiler/luci-interpreter/src/kernels/FullyConnected.test.cpp

```diff
@@ -26,26 +26,54 @@ namespace
 
 using namespace testing;
 
-TEST(FullyConnectedTest, Float)
+// Returns the corresponding Bias DataType given the type T(Input Tensor Type).
+template <typename T> constexpr DataType getBiasType()
 {
-  Shape input_shape{3, 2, 2, 1};
-  std::vector<float> input_data{
-      -3, -5, 5,  4, 9,  -2, // batch = 0
-      -3, -2, -4, 9, -8, 1,  // batch = 1
-  };
-  Shape weights_shape{3, 6};
-  std::vector<float> weights_data{
-      -3, -7, 4, -4, -6, 4,  // unit = 0
-      3,  5,  2, 3,  -3, -8, // unit = 1
-      -3, 7,  4, 9,  0,  -5, // unit = 2
-  };
-  Shape bias_shape{3};
-  std::vector<float> bias_data{-1, -5, -8};
+  if (std::is_same<T, float>::value)
+    return DataType::FLOAT32;
+  if (std::is_same<T, uint8_t>::value)
+    return DataType::S32;
+  return DataType::Unknown;
+}
+
+template <typename T>
+void Check(std::initializer_list<int32_t> input_shape, std::initializer_list<int32_t> weights_shape,
+           std::initializer_list<int32_t> bias_shape, std::initializer_list<int32_t> output_shape,
+           std::initializer_list<float> input_data, std::initializer_list<float> weights_data,
+           std::initializer_list<float> bias_data, std::initializer_list<float> output_data)
+{
+  float kQuantizedTolerance = getTolerance(-127, 128, 255);
+  std::pair<float, int32_t> quant_param = quantizationParams<T>(-127, 128);
+  Tensor input_tensor{
+      getElementType<T>(), input_shape, {{quant_param.first}, {quant_param.second}}, ""};
+  Tensor weights_tensor{
+      getElementType<T>(), weights_shape, {{quant_param.first}, {quant_param.second}}, ""};
+  Tensor bias_tensor{getBiasType<T>(), bias_shape, {}, ""};
```

Comment:
How did you set the quantization params and tolerance?

Reply:
Referenced from TFlite test, but just I reference that could cover all input and output data.
https://github.com/tensorflow/tensorflow/blob/ee598066c4cb31ec5ed3106e61ba99ef004a4bae/tensorflow/lite/kernels/fully_connected_test.cc#L460-L464

Reply:
It seems that qparam of input and weights should be changed; please create another qparam with min/max -63.5/64.

Bias has qparam whose scale is (scale of input) * (scale of weights) and zero point is 0.
(https://github.com/tensorflow/tensorflow/blob/ee598066c4cb31ec5ed3106e61ba99ef004a4bae/tensorflow/lite/kernels/fully_connected_test.cc#L168)

Output uses the current qparam (-127~128), so leave it as is.

Reply:
Okay I will fix it.